src/main/java/org/takes/http/BkBasic.java

```diff
@@ -87,16 +110,44 @@ public void accept(final Socket socket) throws IOException {
     }
 
     /**
+     * Whether to http connection is persistent or not.
+     * @param req RqLive
+     * @return Boolean
+     * @throws IOException if fails
+     */
+    private boolean isConnectionKeepAlive(final RqLive req) throws IOException {
+        boolean keep = false;
+        final RqHeaders.Base rqHeaders = new RqHeaders.Base(req);
```

Comment:
@mstzn36 see http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html